tsconfig.base.json

```diff
@@ -25,6 +25,8 @@
       "@wepublish/consent/editor": ["libs/consent/editor/src/index.ts"],
       "@wepublish/editor/api": ["libs/editor/api/src/index.ts"],
       "@wepublish/editor/api-v2": ["libs/editor/api-v2/src/index.ts"],
+      "@wepublish/events/api": ["libs/events-import/api/src/index.ts"],
```

Comment:
shouldn't be this `@wepublish/events-import/api` or `["libs/events/import/api/src/index.ts"]`?

website-builder i moved to website/builder in my current code